services/graphite/config.go

```diff
@@ -54,6 +54,7 @@ type Config struct {
 	Enabled          bool          `toml:"enabled"`
 	BindAddress      string        `toml:"bind-address"`
 	Database         string        `toml:"database"`
+	RetentionPolicy  string        `toml:"retention-policy"`
```

Comment:
Ah, this requires #6586 to be merged for it to work properly.

I forgot about that. I'll wait until that's merged and rebase on top of it. That PR makes a blank retention policy equal the default retention policy.